_posts/2015-05-23-photoacoustic-penetration.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hearing with light: photoacoustic radiology improved"
+author: RaoOfPhysics
+---
+
+_Using interstitial light source for tissue-imaging technique based on photoacoustic spectroscopy improves penetration depth and shows promise for minimally invasive radiology_
+
+Photoacoustics (PA) involves shining modulated light (*photo–*) on a material, causing the substance to undergo rapid thermal expansion and contraction in such a way that the pressure differences manifest as sound waves (*acoustics*)[^1]. Materials vary in the sounds they produce corresponding to light at different wavelengths. A light-vs-sound spectrum for assorted materials can be created by “listening” to the pressure waves produced in that material by light shone on it at varying wavelengths. The phenomenon was discovered by Alexander Graham Bell in the late 19th Century, but has found use in modern materials science relatively recently.
```

Comment:
The thermal expansion doesn't cause it to give off heat. The energy absorbed from the light heats the material causing it to expand. When the light is cut off, it cools.

Alternating the light on and off causes the material to expand and contract.

I'll rephrase, though. Good point.